extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -239,17 +237,18 @@ public function executeFaviconUpload() {
 		$this->faviconImageName = '';
 		$this->faviconImageUrl = '';
 
-		$status = $this->uploadImage($upload);
-		if($status['status'] === 'uploadattempted' && $status['isGood']) {
-			$file = $upload->getLocalFile(); /* @var $file LocalFile */
+		$status = $this->uploadImage( $upload );
+		if ( $status['status'] === 'uploadattempted' && $status['isGood'] ) {
+			$file = $upload->getLocalFile();
+			/* @var $file LocalFile */
 			$this->faviconImageUrl = wfReplaceImageServer( $file->getUrl() );
 			$this->faviconImageName = $file->getName();
 
 			// if wordmark url is not set then it means there was some problem
 			if ( $this->faviconImageUrl == null ) {
-				$this->errors = array( wfMsg( 'themedesigner-unknown-error' ) );
+				$this->errors = [ wfMsg( 'themedesigner-unknown-error' ) ];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)